libs/api-bot/default.nix

```diff
@@ -0,0 +1,25 @@
+{ mkDerivation, aeson, ansi-terminal, api-client, async, attoparsec
```

Comment:
Can all of these files have a comment stating
1. that this file is generated
2. how this file was generated
3. how to regenerate exactly this file one more time, and when this becomes necessary.

This could also be a comment linking to a longer description on why these files are needed, who calls them, how to regenerate them.

Reply:
All these files have comments now.